docs/languages/powershell.md

```diff
@@ -0,0 +1,209 @@
+# Editing PowerShell in Visual Studio Code
+
+PowerShell is a task-based command line shell and scripting language built on .NET .
+
+PowerShell provides a powerful toolset for administrators on any platform.
+
+[The PowerShell Extension](https://github.com/PowerShell/vscode-powershell) provides rich language Support and IDE capabilities such as completions, definition tracking and linting analysis for PowerShell versions 3, 4, 5 and 5.1 as well as all versions of PowerShell Core.
+
+## Install the PowerShell Extension on Visual Studio Code
+
+The official PowerShell Extension can be installed by following the steps in the
+[Visual Studio Code Docs](https://code.visualstudio.com/docs/editor/extension-gallery)
+or by going directly to the Marketplace and clicking the [Install Button](vscode:extension/ms-vscode.PowerShell)
+
+You can also install the PowerShell Extension from within Visual Studio Code by
+browsing the Extension Marketplace under the Marketplace view in the Activity Bar for extensions.
+You can access this with the shortcut <kbd>ctrl</kbd>-<kbd>Shift</kbd>-<kbd>x</kbd> or by clicking on the
```

Comment:
nitpick: add the macOS shortcut in parenthesis:

"<kbd>ctrl</kbd>-<kbd>Shift</kbd>-<kbd>x</kbd> ( or <kbd>cmd</kbd>-<kbd>Shift</kbd>-<kbd>x</kbd> on macOS)"